src/backend/joanie/settings.py

```diff
@@ -795,6 +795,32 @@ class Production(Base):
         },
     }
 
+    # Cache
+    # Enable the alternate connection factory.
+    DJANGO_REDIS_CONNECTION_FACTORY = values.Value(
+        "django_redis.pool.ConnectionFactory",
+        environ_prefix=None,
+        environ_name="DJANGO_REDIS_CONNECTION_FACTORY",
+    )
+
+    CACHES = {
+        "default": {
+            "BACKEND": values.Value(
+                "django_redis.cache.RedisCache", environ_name="CACHE_DEFAULT_BACKEND"
+            ),
+            # The hostname in LOCATION is the primary (service / master) name
+            "LOCATION": values.Value(
+                "redis://service_name/db", environ_name="CACHE_DEFAULT_LOCATION"
```

Comment:
In order to have a working configuration for the service `app` for local production test, I'll use the name of the redis service.

```suggestion
                "redis://redis/db", environ_name="CACHE_DEFAULT_LOCATION"
```